README.md

```diff
@@ -111,6 +111,13 @@ WARNING: Because users are sending their own credentials to the proxy, it's impo
 setting only when the proxy is under control of the cluster administrators. Otherwise, end users 
 may be unwittingly provide their credentials to untrusted components that can then act as them.
 
+When configuring for delegation by default `X-Forwarded-Access-Token` header will not be set on
+the upstream request if you wish to pass the bearer token received from the client upstream, in
+addition to `--openshift-delegate-urls`,  `--pass-user-bearer-token` option needs to be provided.
+
```

Comment:
When configured for delegation, Oauth Proxy will not set the `X-Forwarded-Access-Token` header on the upstream request. If you wish to forward the bearer token received from the client, you will have to use the `--pass-user-bearer-token` option in addition to `--openshift-delegate-urls`.